tests/Api/Shop/GetCountriesTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Tests\Api\Shop;
+
+use Sylius\Tests\Api\JsonApiTestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+final class GetCountriesTest extends JsonApiTestCase
```

Comment:
I'm wondering, do we want to have tests for different operations for the same resource separated between classes or functions in one class? If the latter, I would name it `CountriesTest` and name the function `it_returns_countries_list`/`it_provides_countries_list` or something like it 🖖